src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ConstructorInitializerSymbolReferenceFinder.cs

```diff
@@ -76,22 +79,78 @@ protected override bool CanFind(IMethodSymbol symbol)
                 {
                     return true;
                 }
+                else if (methodSymbol.Parameters.IsDefaultOrEmpty
+                    && syntaxFactsService.IsConstructorToken(t))
```

Comment:
Oh.  MethodSymbol is for the BaseConstructor?  If that's the case, what about:

c#
class Base { 
     Base(int i = 0) { }
}

class Derived {
    Derived(string s) { }
}
```

?